<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnImg1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>input 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImg2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>input 2</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="result">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>531</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="img1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="img2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExec">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>execute</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
